bot/qwen_code.py

```
@@ -0,0 +1,376 @@
+            _log.info("[ComfyTV/bot] Added comfytv MCP server: %s", mcp_endpoint)
+        except Exception as e:
+            _log.warning("[ComfyTV/bot] Failed to configure MCP server: %s", e)
+
+    def _build_argv(self, turn: TurnRequest) -> list[str]:
+        argv = resolve_qwen_command()
+        if not argv:
+            raise RuntimeError("qwen executable not found")
+        argv = argv + [
+            "-p", turn.user_text,
+            "--output-format", "stream-json",
+        ]
```

Ruff (0.16.1)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)